quiche/src/recovery/gcongestion/bbr2/startup.rs

```diff
@@ -85,6 +85,7 @@ impl ModeImpl for Startup {
         let check_persisten_queue =
             params.max_startup_queue_rounds > 0 && !has_bandwidth_growth;
         if check_persisten_queue {
+            // https://github.com/google/quiche/blob/27eca0257490df89d2bd2c2a8bcea15565e7831c/quiche/quic/core/congestion_control/bbr2_startup.cc#L60-L62
```

Comment:
What branch is 27eca0257490df89d2bd2c2a8bcea15565e7831c associated with?  I see it on github, but it is not a revision that my local git clients know about.

Reply:
Its just a permalink to google quiche to avoid code drift.